sql-statements/sql-statement-admin-show-ddl.md

```diff
@@ -80,13 +80,17 @@ The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the curre
 - `START_TIME`: the start time of the DDL operation.
 - `END_TIME`: the end time of the DDL operation.
 - `STATE`: the state of the DDL operation. Common states include the following:
+    - `none`: indicates that the operation has not started yet.
     - `queueing`: indicates that the operation job has entered the DDL job queue but has not been executed because it is still waiting for an earlier DDL job to complete. Another reason might be that after executing the `DROP` operation, it will become the `none` state, but it will soon be updated to the `synced` state, indicating that all TiDB instances have been synchronized to that state.
```

Comment:
```suggestion
    - `queueing`: indicates that the operation job has entered the DDL job queue but has not been executed because it is still waiting for an earlier DDL job to complete. Another reason might be that after executing the `DROP` operation, it will become the `done` state, but it will soon be updated to the `synced` state, indicating that all TiDB instances have been synchronized to that state.
```

Is this a typo or is it correct?

Reply:
@wjhuang2016 PTAL